pkg/crc/cluster/clusteroperator.go

```diff
@@ -145,6 +147,37 @@ func getStatus(ctx context.Context, lister operatorLister, selector []string) (*
 	return cs, nil
 }
 
+func GetClusterNodeStatus(ctx context.Context, ip string, kubeconfigFilePath string) (*Status, error) {
+	status := &Status{
+		Available: true,
+	}
+	clientSet, err := kubernetesClient(ip, kubeconfigFilePath)
+	if err != nil {
+		return nil, err
+	}
+	nodes, err := clientSet.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
```

Comment:
i guess you already tried to use the `openshift` client-go and it didn't work with that and needed the kubernetes `clientset`, do you know why from the openshift clientset we can't get the node statuses?

Reply:
I had the same question, but a clientset is apparently a thin wrapper over a set o f HTTP APIs. So `openshift/config/clientset` wraps mostly openshift's `ConfigV1()` API, and `k8s/clientset` wraps a different set of HTTP APIs (https://github.com/kubernetes/client-go/blob/master/kubernetes/clientset.go )